src/components/ArtistList.md

```diff
@@ -0,0 +1,11 @@
+```js
+<ArtistList
+  artists={[
+    {
+      artist: 'Dance Gavin Dance',
+      songTitle: 'We Own The Night',
+      id: 'jkas872301jklas',
```

Comment:
I think, you don't have to hand in the id at for the Styleguidist, but I'm not too sure.. :)

Reply:
So it's complete for me, even if it's not rendered.